src/analysis/special_functions/sqrt.lean

```diff
@@ -0,0 +1,163 @@
+/-
+Copyright (c) 2021 Yury G. Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Yury G. Kudryashov
+-/
+import data.real.sqrt
+import analysis.calculus.inverse
+import measure_theory.borel_space
+
+/-!
+# Smoothness of `real.sqrt`
+
+In this file we prove that `real.sqrt` is infinitely smooth at all points `x ≠ 0` and provide some
+dot-notation lemmas.
+
+## Tags
+
+sqrt, differentiable
+-/
+
+open set
+open_locale topological_space
+
+namespace real
+
+/-- Local homeomorph between `(0, +∞)` and `(0, +∞)` with `to_fun = λ x, x ^ 2` and
+`inv_fun = sqrt`. -/
+noncomputable def sqr_local_homeomorph : local_homeomorph ℝ ℝ :=
```

Comment:
I thought the consensus was that we should use `sq` in lemma names for the square, see https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/sqr.20vs.20sq.20vs.20pow_two/near/224972795

Reply:
Thanks! I forgot about this.